File: app/src/main/java/vn/loitp/app/activity/customviews/wwlvideo/interfaces/FragmentHost.java
package vn.loitp.app.activity.customviews.wwlvideo.interfaces;

import vn.loitp.app.activity.customviews.wwlvideo.utils.WWLVideoDataset;

/**
 * Created by thangn on 2/26/17.
 */

public interface FragmentHost {
    void goToDetail(WWLVideoDataset.DatasetItem item);

    void onVideoCollapse();

    void onVideoFullscreen(boolean selected);
}

File: app/src/main/java/vn/loitp/app/activity/customviews/wwlVideo/interfaces/FragmentHost.kt
package vn.loitp.app.activity.customviews.wwlVideo.interfaces

import vn.loitp.app.activity.customviews.wwlVideo.utils.WWLVideoDataset

interface FragmentHost {
    fun goToDetail(item: WWLVideoDataset.DatasetItem)

    fun onVideoCollapse()

    fun onVideoFullscreen(selected: Boolean)
}
